### ISSUE TITLE
Fails after 0.6.1 update

### ISSUE THREAD

**Issue by sobolevn** [mgaitan/sphinxcontrib-mermaid#66]
Link: https://github.com/wemake-services/wemake-python-styleguide/pull/1859
Job: https://readthedocs.org/projects/wemake-python-styleguide/builds/12971370/

Traceback: 

```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/application.py", line 352, in build
    self.builder.build_update()
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 298, in build_update
    len(to_build))
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 360, in build
    self.write(docnames, list(updated_docnames), method)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 534, in write
    self._write_serial(sorted(docnames))
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 544, in _write_serial
    self.write_doc(docname, doctree)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/builders/html/__init__.py", line 597, in write_doc
    self.docwriter.write(doctree, destination)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/docutils/writers/__init__.py", line 78, in write
    self.translate()
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/writers/html.py", line 71, in translate
    self.document.walkabout(visitor)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/docutils/nodes.py", line 214, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/docutils/nodes.py", line 214, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/docutils/nodes.py", line 214, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/docutils/nodes.py", line 206, in walkabout
    visitor.dispatch_visit(self)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/util/docutils.py", line 468, in dispatch_visit
    method(node)
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinxcontrib/mermaid.py", line 267, in html_visit_mermaid
    render_mm_html(self, node, node['code'], node['options'])
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinxcontrib/mermaid.py", line 228, in render_mm_html
    _fmt = self.builder.config.mermaid_output__fmt
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/config.py", line 257, in __getattr__
    raise AttributeError(__('No such config value: %s') % name)
AttributeError: No such config value: mermaid_output__fmt

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/wemake-python-styleguide/envs/1859/lib/python3.7/site-packages/sphinx/config.py", line 257, in __getattr__
    raise AttributeError(__('No such config value: %s') % name)
AttributeError: No such config value: mermaid_output__fmt
```

**Comment by SchindlerTo** [comment]
Happy users so far, thanks for that!
Unfortunately, we have the same issue in our build pipeline, which fails with mermaid 0.6.1:
We tried it using python 3.9, same result.

`Exception occurred:
  File "/usr/local/lib/python3.8/site-packages/sphinx/config.py", line 257, in __getattr__
    raise AttributeError(__('No such config value: %s') % name)
AttributeError: No such config value: mermaid_output__fmt
The full traceback has been saved in /tmp/sphinx-err-_qb80bhs.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:26: html] Error 2`

**Comment by gil-obradors** [comment]
Seems that there's a PR for this #65

**Comment by AlparslanErol** [comment]
I had the same issue as well and solved the problem...
I simply freeze the sphinxcontrib-mermaid as like previous version in my project config.py file.

**Comment by gil-obradors** [comment]
Solved at  0.6.2

You can close this issue
